core/templates/dev/head/pages/base.html

```diff
@@ -31,7 +31,7 @@
 {%- endmacro %}
 
 <!DOCTYPE html>
-<html ng-app="oppia" ng-controller="Base" itemscope itemtype="http://schema.org/Organization">
+<html ng-app="oppia" update-language lang="en" ng-controller="Base" itemscope itemtype="http://schema.org/Organization">
```

Comment:
I tried this just now. On empty cache and hard reload, the lang attribute changes to en for a second or so before changing to the current language.